docker-compose.yml

```diff
@@ -37,6 +37,47 @@ services:
     environment:
       KS_SWIFT_PUBLIC_URL: http://127.0.0.1:49177
 
+  graylog_mongo:
+    container_name: mongo
+    image: mongo:4.2
+    networks:
+      - default
+
+  graylog_elasticsearch:
+    container_name: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.2
+    environment:
+      - http.host=0.0.0.0
+      - transport.host=localhost
+      - network.host=0.0.0.0
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    ports:
+      - 9300:9300
+    deploy:
+      resources:
+        limits:
+          memory: 1g
+    networks:
+      - default
```

Comment:
Can't we use the existing elasticsearch service to save local resources?

Reply:
If we test a transfer from ldp to es, it will fail isn't it?

Reply:
It has to be tested, but I don't think so: ES is a backend used by graylog but graylog will be targeted by our requests.